templates/cisco_xr_admin_show_environment_fan.template

```diff
@@ -0,0 +1,39 @@
+Value MODULE (0\/(FT|PM)\d)
```

Comment:
what is wrong with `/S+` here?

Reply:
@itdependsnetworks  I though the regex should be specific if possible. Is that not correct?

Reply:
To an extent yes, to an extent no :) 

Looking at the data, it doesn't seem like that would cover all future use cases. You are fairly well protected here in that you properly transition state to `FM`, after which I think it's a safe assumption that this is the only thing that would fit, and not be a false positive.